.github/workflows/advanced.yml

```diff
@@ -421,6 +421,11 @@ jobs:
           path: ${{ env.ARTIFACT_PATH }}/${{ env.TESTS_PACKAGE_NAME }}
           retention-days: 3
 
+      # This test builds and also runs on Ubuntu as it doesn't need to run any executable on Windows
+      - name: Test PE/COFF big object format
+        run: |
+          .github/scripts/tests/test-gas-big-obj.sh
```

Comment:
I don't think it's sustainable from a long term perspective to add a new job for every test case we might be adding. I'd prefer if this test case would be executen within [https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/blob/main/.github/scripts/tests/execute-tests.sh](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/blob/main/.github/scripts/tests/execute-tests.sh) step.

Reply:
It cannot be used in execite-tests.sh as it is used in an Arm64 host, however the idea is correct and probably should be implemented in tests/test-toolchain.sh

Reply:
Yes, I meant the `tests/test-toolchain.sh`. Sorry for the miss-information.

Reply:
Created `tests/test-toolchain.sh` as a place for more tests in the future